src/Components/App/App.js

```diff
@@ -38,37 +40,46 @@ const App = () => {
 
   return (
     <section className="App">
-      <Header />
-      <Switch>
-        <Route path="/about" render={() => <About />} />
-        <Route path="/styles" render={() => <Styles />} />
-        <Route path="/contact" render={() => <ContactForm />} />
-        <Route
-          path="/stylist/:id"
-          render={({ match }) => {
-            const { id } = match.params;
-            return <StylistPage id={id} />;
-          }}
-        />
-        <Route
-          exact
-          path="/"
-          render={() => (
-            <>
-              <SearchForm
-                retrieveSearchResults={retrieveSearchResults}
-                setIsLoading={setIsLoading}
-              />
-              <SearchResultsContainer
-                results={results}
-                style={style}
-                isLoading={isLoading}
-              />
-            </>
-          )}
-        ></Route>
-      </Switch>
-      <Footer />
+      {!isModalOpen && (
```

Comment:
If the modal is not open, displays the regular pages with a header and footer as normal.